src/config/typeorm.config.ts

```diff
@@ -1,15 +1,15 @@
-import { TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { DataSource, DataSourceOptions } from 'typeorm';
 
-const typeOrmConfig: TypeOrmModuleOptions = {
+export const dataSourceOptions: DataSourceOptions = {
```

Comment:
Why did you change export and type?

Reply:
The documentation for the latest typeorm version recommends to configure database connection in this way. Also, migrations work only if data-source file contain export of DataSource instance.
![err](https://user-images.githubusercontent.com/31353437/232166942-f96bb62e-c7d6-4c40-8136-40109d9e118a.jpg)

Reply:
Ok.